bentoml/_internal/bento/build_config.py

```diff
@@ -147,52 +95,24 @@ class DockerOptions:
             attr.validators.in_(DOCKER_SUPPORTED_DISTROS)
         ),
     )
-
     python_version: str = attr.field(
         converter=_convert_python_version,
         default=None,
         validator=attr.validators.optional(
             attr.validators.in_(SUPPORTED_PYTHON_VERSIONS)
         ),
     )
-
     cuda_version: t.Union[str, Literal["default"]] = attr.field(
         default=None,
         converter=_convert_cuda_version,
         validator=attr.validators.optional(
             attr.validators.in_(SUPPORTED_CUDA_VERSIONS.values())
         ),
     )
-
-    # A user-provided environment variable to be passed to a given bento
-    # accepts the following format:
-    #
-    # env:
-    #  - ENVAR=value1
-    #  - FOO=value2
-    #
-    # env:
-    #  ENVAR: value1"Options"
-    #  FOO: value2
-    #
-    #  env: /path/to/.env
-    env: t.Dict[str, t.Any] = attr.field(
-        default=None,
-        converter=_convert_user_envars,
-        validator=attr.validators.optional(_envars_validator),
-    )
-
-    # A user-provided system packages that can be installed for a given bento
-    # using distro package manager.
+    env: t.Optional[t.Dict[str, t.Any]] = None
     system_packages: t.Optional[t.List[str]] = None
-
-    # A python or sh script that executes during docker build time
     setup_script: t.Optional[str] = None
-
-    # A user-provided custom docker image
     base_image: t.Optional[str] = None
-
-    # A user-provided dockerfile jinja2 template
     dockerfile_template: str = attr.field(
         default=None,
         validator=attr.validators.optional(
```

Comment:
this will fail if Dockerfiles is provided as a relative path and user provided a build_ctx